app/src/main/java/com/example/android/weatherapp/utilities/Helper.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.android.weatherapp.utilities
+
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+import kotlin.math.round
+
+private const val DATE_FORMAT = "MMMM dd, yyyy"
+
+object Helper {
```

Comment:
Replace object with class

Reply:
Created class in the next commit of this branch